### PR TITLE
Add fee parameter to transaction signing guide

### PR DESCRIPTION
**Pull request by markmhx**
Per https://forum.stacks.org/t/mempool-congestion-on-stacks-observations-and-next-steps-from-hiro/12325/5

### CODE REVIEW

**Comment by markmhx** [comment]
@fbwoolf @kyranjamie The main thing here is to confirm whether the parameter is microstacks or regular stacks, I think?

File: src/pages/build-apps/guides/transaction-signing.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ interface STXTransferOptions {
   amount: string;
   memo?: string;
   network: StacksNetwork;
+  fee: number,
```
Comment (fbwoolf) [inline src/pages/build-apps/guides/transaction-signing.md:93]: This can be a number or a string.

Comment (markmhx) [reply src/pages/build-apps/guides/transaction-signing.md:93]: String would be something like "3" for "3 STX" or "3000000" for "3 STX" (in microstacks)?

Comment (kyranjamie) [reply src/pages/build-apps/guides/transaction-signing.md:93]: Always µSTX units

**Review by fbwoolf — APPROVED** [review]

**Review by kyranjamie — APPROVED** [review]

**Review by pgray-hiro — APPROVED** [review]

**Pull request merged**